llvm/test/tools/llvm-readobj/ELF/decompress.test

```diff
@@ -0,0 +1,67 @@
+# REQUIRES: zlib
```

Comment:
Perhaps worth naming decompress-zlib.test, to allow for a zstd version too? (Same applies for the unsupported case).

Reply:
Sounds good. Done. Added a zstd version.

There is `decompress-zlib-unsupported.test` but no `decompress-zstd-unsupported.test`. The latter probably doesn't add much coverage.

Reply:
That's fine, but one thought that did occur to me is that I suspect zlib is available on more machines than zstd, therefore by having a zstd unsupported test case, you'd be more likely to have coverage of both the supported and unsupported code paths on the same machine.

I'm okay if you still want to do it though.

Reply:
I think Windows is a popular OS that often misses zlib support. However, "often" may degrade to "sometimes". Anyway, I just added `decompress-zstd-unsupported.test`...